docs/pages/monitoring/nuts-go.rst

```diff
@@ -28,10 +28,173 @@ It'll return some text displaying the current status of the various services
 
 .. code-block:: text
 
-Status
-	Registered engines: Status,Logging,Crypto,Registry,Events octopus,ConsentLogicInstance,ConsentStore,Validation,Auth,ConsentBridgeClient
-Logging
-	verbosity:
-Events octopus
-	Nats streaming server: mode: STANDALONE @ 0.0.0.0:4222, ID: nuts, last error: NONE
-	DB: ping: true
+    Status
+        Registered engines: Status,Logging,Crypto,Registry,Events octopus,ConsentLogicInstance,ConsentStore,Validation,Auth,ConsentBridgeClient
+    Logging
+        verbosity:
+    Events octopus
+        Nats streaming server: mode: STANDALONE @ 0.0.0.0:4222, ID: nuts, last error: NONE
+        DB: ping: true
+
+Metrics
+*******
+
+The Nuts service executable has build-in support for **Prometheus**. Prometheus is a time-series database which supports a wide variety of services. It also allows for exporting metrics to different visualization solutions like **Grafana**. See https://prometheus.io/ for more information on how to run Prometheus. The metrics are exposed at ``/metrics``
+
+Configuration
+=============
+
+In order for metrics to be gathered by Prometheus. A ``job`` has to be added to the ``prometheus.yml`` configuration file.
+Below is a minimal configuration file that will only gather Nuts metrics:
+
+.. code-block:: yaml
+
+    # my global config
+    global:
+      scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+      evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+      # scrape_timeout is set to the global default (10s).
+
+    # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+    rule_files:
+    # - "first_rules.yml"
+    # - "second_rules.yml"
+
+    # A scrape configuration containing exactly one endpoint to scrape:
+    # Here it's Prometheus itself.
+    scrape_configs:
+      # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+      - job_name: 'nuts'
+        metrics_path: '/metrics'
+        scrape_interval: 5s
+        static_configs:
+          - targets: ['127.0.0.1:1323']
+
+It's imported to enter the correct IP/domain and port where the Nuts node can be found!
+
+Exported metrics
+================
+
+The Nuts service executable exports the following metrics by default. These cover the basic needs for monitoring the process and http layer.
```

Comment:
Is it useful to include this here or can we refer to an external reference so we don't have to keep this up to date?

Reply:
I couldn't find a direct reference to a curated list.